lib/commands/release/index.ts

```diff
@@ -95,25 +99,29 @@ export default class ReleaseCmd extends Command {
 		];
 
 		const release = await balena.models.release.get(commitOrId, {
-			$select: fields,
+			...(!options.json && { $select: fields }),
```

Comment:
I think we shouldn't just select all fields by default with JSON, we need to explicitely $select the fields we want to avoidi the output of the CLI changing by a change on the API. I would be ok with something like: 

```
const fields: Array<keyof BalenaSdk.Release> = [
    'id',
    'commit',
    'created_at',
    'status',
    'semver',
    'is_final',
    'build_log',
    'start_timestamp',
    'end_timestamp',
];

const jsonFields = [...tableFields, 'note', 'is_final'....]; // it would be interesting to get from an user perspective what fields you care about here.

...
$select: options.json ? jsonFields : tableFields,
```

@thgreasi what do you think?

Reply:
I will do this if required, but I don't think it makes sense for the use case of printing the json result of the command. 

With tools that can return results in json format, it is standard practice to return all of the data, like `docker inspect`, `kubectl get pods -o json`, `cargo workspaces list --json`, `aws s3api list-objects`, etc.

The primary value of offering API access is allowing consumers to find solutions for their challenges without requiring the provider to guess at all possible use cases for it. Additionally, the consumers will not know it is being filtered, and may need to solve a problem with data that is being hidden and not know they can access it.

By filtering the data it would be arbitrarily restricting the use of the returned data.